cfme-performance/conftest.py

```diff
@@ -148,6 +149,7 @@ def pytest_sessionfinish(session, exitstatus):
     summary = ', '.join(results)
     logger().info(log.format_marker('Finished test run', mark='='))
     logger().info(log.format_marker(str(summary), mark='='))
+    generate_version_files()
```

Comment:
I would prefer we specify the fixture where the tests are such as:

https://github.com/akrzos/cfme_tests/blob/performance/cfme/tests/perf/benchmark/test_capacity_and_utilization.py#L20-L22